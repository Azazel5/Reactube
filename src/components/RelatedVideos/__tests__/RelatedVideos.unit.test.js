import React from 'react'
import {shallow} from 'enzyme'
import RelatedVideos from '../RelatedVideos'

describe('<RelatedVideos />', () => {
    test('renders <RelatedVideos />', () => {
        const wrapper = shallow(<RelatedVideos />)
        expect(wrapper).toMatchSnapshot()
    })
})