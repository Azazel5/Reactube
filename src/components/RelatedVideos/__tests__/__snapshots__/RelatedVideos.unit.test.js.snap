// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RelatedVideos /> renders <RelatedVideos /> 1`] = `
<div
  className="related-videos"
>
  <nextUpVideo />
  <videoPreview
    horizontal={true}
  />
  <videoPreview
    horizontal={true}
  />
  <videoPreview
    horizontal={true}
  />
</div>
`;
